lib/collections/phoneNumber.js

```diff
@@ -0,0 +1,87 @@
+phoneNumber = new Mongo.Collection('sites1');
```

Comment:
The variable should start from capital letter, while the argument should be with small letter. Takecare of it next time.